qpylib/encdec.py

```diff
@@ -98,7 +98,7 @@ def __encrypt_string(self, clear_text_string):
             self.config[self.name]['ivz'].encode('utf-8'),
             segment_size=128)
         clear_text_padded_string = self.__pad_string(clear_text_string)
-        encrypted_bytes = aes.encrypt(clear_text_padded_string)
+        encrypted_bytes = aes.encrypt(clear_text_padded_string.encode("utf8"))
```

Comment:
Is the code broken without the call to encode("utf8") ?

Reply:
It seemed to be for me. On my windows laptop I couldn't install pycrypto to start with. Dropped in pycryptodome and the only error was in this area, to the effect of "can't pass 'str' to c". Stack Overflow indicated that the encoding likely needed to be changed. 

Reproduced the same on a coworkers mac. They installed pycrypto, worked fine, switched to cryptodome, same error. There may be a more elegant solution I did not find.